nudging/dataset/base.py

```diff
@@ -7,25 +7,29 @@
 class BaseDataSet(ABC):
     """Base class for DataSet """
 
-    nudge_type = None
```

Comment:
Removing nudge_type and nudge_domain as Dataset attributes does bot work for real datasets in the current setup